engine/table/src/main/java/io/deephaven/engine/table/impl/locations/impl/CompositeTableDataService.java

```diff
@@ -53,6 +53,37 @@ public CompositeTableDataService(@NotNull String name, @NotNull final ServiceSel
         this.serviceSelector = Require.neqNull(serviceSelector, "serviceSelector");
     }
 
+    @Override
+    @Nullable
+    public TableLocationProvider getRawTableLocationProvider(@NotNull TableKey tableKey,
+            @NotNull TableLocationKey tableLocationKey) {
```

Comment:
is the coding standard in core not to use 'final' on parameters (and local variables)?

Reply:
Looking at nearby code, it seems like the pattern is there. But IJ warnings didn't manifest for me, which I'm guessing is related to me neglecting the `final`s. I've added them now.